spec/features/actors_feature_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+RSpec.describe "Actors", type: :feature do
+
+  before do
+    scorsese = create(:director, name: "Martin Scorsese", dob: "November 17, 1942")
+    departed = create(:movie, title: "The Departed", year: 2006, duration: 151, director: scorsese)
+    goodfellas = create(:movie, title: "Goodfellas", year: 1990, duration: 146, director: scorsese)
+
+    nolan = create(:director, name: "Christopher Nolan", dob: "July 30, 1970")
+    dark_knight = create(:movie, title: "The Dark Knight", year: 2008, duration: 152, director: nolan)
+    inception = create(:movie, title: "Inception", year: 2010, duration: 148, director: nolan)
+
+    leo = create(:actor, name: "Leonardo DiCaprio", dob: "November 11, 1974")
+    create(:character, name: 'Cobb', movie: inception, actor: leo)
+    create(:character, name: 'Billy Costigan', movie: departed, actor: leo)
+
+    jack = create(:actor, name: "Jack Nicholson", dob: "April 22, 1937")
+    create(:character, name: 'Frank Costello', movie: departed, actor: jack)
+
+    bob = create(:actor, name: "Robert De Niro", dob: "August 17, 1943")
+    create(:character, name: 'James Conway', movie: goodfellas, actor: bob)
+  end
+
+  context "index page" do
+    # TODO
+
+    it "displays each actor's name and dob", points: 5 do
+      visit "/actors"
+
+      actors = Actor.all
+      actors.each do |actor|
+        expect(page).to have_content(actor.name)
+        expect(page).to have_content(actor.dob)
+      end
+    end
+
+    it "displays a functional delete link for each actor", points: 5 do
+      visit "/actors"
```

Comment:
In general, a lot of these tests are testing `starter_generators` Golden Seven functionality. Should we bother, outside of perhaps testing that they generated the right table/column names?

I am working on very detailed tests of Golden Seven in [this branch of Photogram G7](https://github.com/appdevwinter17/photogram_golden_seven/tree/solutions). We should be able to just copy-paste those over, or even just build them into the generator.

In this project, should we just concentrate on association- and validation-specific tasks?